provision.yaml

```diff
@@ -2,3 +2,6 @@
 debian:
   provisioner: docker
   images: ['litmusimage/debian:10']
+centos:
+  provisioner: docker
+  images: ['litmusimage/centos:7', 'litmusimage/centos:8']
```

Comment:
The provision.yaml is deprecated, since tests have been moved to GHA. You don't need to change that, I just forgot to clean up.

Reply:
I removed this change